static/js/tools/map/chart_loader.tsx

```diff
@@ -132,7 +136,26 @@ export function ChartLoader(): JSX.Element {
   //   };
   // }, []);
 
+  function sourcesReducer(
```

Comment:
wonder if we should have these reducers in a separate file like with chartStoreReducer?